centaurCwlRunner/src/main/scala/centaur/cwl/CentaurCwlRunner.scala

```diff
@@ -0,0 +1,161 @@
+package centaur.cwl
+
+import better.files._
+import centaur.api.CentaurCromwellClient
+import centaur.test.TestOptions
+import centaur.test.standard.{CentaurTestCase, CentaurTestFormat}
+import centaur.test.submit.{SubmitHttpResponse, SubmitWorkflowResponse}
+import centaur.test.workflow.{AllBackendsRequired, Workflow, WorkflowData}
+import com.typesafe.config.ConfigFactory
+import cromwell.api.model.{Aborted, Failed, NonTerminalStatus, Succeeded}
+import spray.json._
+
+/**
+  * Runs workflows in a "cwl-runner" friendly way.
+  *
+  * https://github.com/broadinstitute/cromwell/issues/2590
+  * https://github.com/common-workflow-language/common-workflow-language/blob/v1.0.1/CONFORMANCE_TESTS.md
+  * https://github.com/common-workflow-language/common-workflow-language/blob/v1.0.1/draft-3/cwl-runner.cwl#L5-L68
+  * https://github.com/common-workflow-language/common-workflow-language/pull/278/files#diff-ee814a9c027fc9750beb075c283a973cR49
+  */
+object CentaurCwlRunner {
+
+  case class CommandLineArguments(workflowSource: Option[File] = None,
+                                  workflowInputs: Option[File] = None,
+                                  quiet: Boolean = false,
+                                  outdir: Option[File] = None)
+
+  // TODO: This would be cleaner with Enumeratum
+  object ExitCode extends Enumeration {
+
+    protected case class Val(status: Int) extends super.Val
+
+    implicit class ValueToVal(val exitCodeValue: Value) extends AnyVal {
+      def status: Int = exitCodeValue.asInstanceOf[Val].status
+    }
+
+    val Success = Val(0)
+    val Failure = Val(1)
+    val NotImplemented = Val(33)
+  }
+
+  private val parser = buildParser()
+  private lazy val centaurVersion = ConfigFactory.load("cromwell-version.conf").getString("version.cromwell")
```

Comment:
The version is "required" by the CWL conformance test suite.

This does read strangely, but is easy to clean up in the future. I didn't want to change it yet. Specifically, each artifact could write _its_ version instead of the "cromwell", aka "root" version.

Right now, the two artifacts that are writing versions are "cromwell-engine" and "centaur-cwl-runner". I didn't want to change too much at the moment to start reading from `cromwell-engine-version.conf`, but perhaps in a future PR.